js/src/OrdinalScaleModel.ts

```diff
@@ -26,6 +26,10 @@ export class OrdinalScaleModel extends ScaleModel {
         };
     }
 
+    typed(values) {
+        return new Int32Array(values.map(Number));
```

Comment:
Is it true that it's an Int32Array?

Couldn't the array be something like `['A', 'B', 'C', 'D']`

Reply:
Ok, we're talking about the domain here, which in the case of ordinal is integer, maybe rename it to `typedRange(values)` ?

Reply:
(typo, domain->range)